.github/workflows/spark_examples_test.yaml

```diff
@@ -0,0 +1,52 @@
+name: "Delta Spark Local Publishing and Examples Compilation"
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        # These Scala versions must match those in the build.sbt
+        scala: [2.12.18, 2.13.13]
```

Comment:
is one not enough?

Reply:
We run spark tests _tests_ for both scala 2.12 and 2.13 ... I see no reason not do use both scala versions just to _compile_ the examples.

If you feel strongly about this, let me know

Reply:
The number of CI jobs is going up. I guess we can revisit this as part of the test CI overhaul.